zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/metrics/TtlKeyCache.java

```diff
@@ -54,6 +54,20 @@ void store(final long key, final long timestamp) {
     timestampToKeys.computeIfAbsent(timestamp, ignored -> new LongArrayList()).add(key);
   }
 
+  long get(final long key) {
+    final var timestamp = keyToTimestamp.get(key);
+    final var keys = timestampToKeys.get(timestamp);
+
+    if (keys != null) {
+      keys.remove(key);
+      if (keys.isEmpty()) {
+        timestampToKeys.remove(timestamp);
```

Comment:
:question: Why do you have to remove the key and timestamp?

Reply:
Because my head was somewhere else :see_no_evil: